keen/tests/client_tests.py

```diff
@@ -492,6 +495,25 @@ def test_interval(self, get):
         resp = keen.count("query test", timeframe="this_2_days", interval="daily")
         self.assertEqual(type(resp), list)
 
+    def test_order_by(self, get):
```

Comment:
Do you plan to add more tests, like making sure it fails for invalid combinations of `limit`, `group_by` and `order_by` or malformed/invalid `limit` or `order_by` params?

Reply:
Heh, I need to write more; yes. I'll do that. Thanks for the (much needed) prod.